client/src/components/AddGames/index.js

```diff
@@ -87,6 +87,18 @@ class AddGames extends React.Component {
         }
 
         gameLogic.addGame(gameObj);
+        this.context.update({
+            gameName: "",
+            minPlayers: "",
+            maxPlayers: "",
+            minPlaytime: "",
+            maxPlaytime: "",
+            minAge: "",
+            rating: "",
+            rules: "",
+            image: "",
+            complexity: "",
+        });
```

Comment:
i think we need to review this and have this be inside a `.then` on line 89. I'm afraid this could create a race condition and unset the context before it should. if i'm wrong, just let me know.